src/content/configuration/node.md

```diff
@@ -16,7 +16,7 @@ This feature is provided by webpack's internal [`NodeStuffPlugin`](https://githu
 
 ## `node`
 
-`object`
+`boolean | object`
 
 This is an object where each property is the name of a Node global or module and each value may be one of the following...
```

Comment:
Please update this too since it can accept bool :)

Reply:
The documentation of the `false` value is at the end:
> Since webpack 3.0.0, the `node` option may be set to `false` to completely turn off the `NodeStuffPlugin` and `NodeSourcePlugin` plugins.

Would you like me to put that up front instead?

Reply:
Ah, missed that one, this looks good! ✨